l10n_br_account_product/models/account_invoice.py

```diff
@@ -756,6 +757,21 @@ def _compute_price(self):
     freight_value = fields.Float(
         'Frete', digits=dp.get_precision('Account'), default=0.00)
     fiscal_comment = fields.Text(u'Observação Fiscal')
+    xped = fields.Char(
+        string=u"Código do Pedido (xPed)",
+        size=15,
+    )
+    nitemped = fields.Char(
+        string=u"Item do Pedido (nItemPed)",
+    )
+
+    @api.onchange("nitemped")
```

Comment:
I tried using `field.Integer`, but it defaults `0` instead of being empty.

The `nItemPed` tag is explicitly optional in the NFe definition. Having the `nitemped` field default to `0` would have the consequence of there always being an `nItemPed' tag in the XML.

There is nothing special about `nitemped`, but it would be unfriendly to wait until the user tries to save the object to notify about the wrong value of what is essentially a very optional field, hidden behind an "Extra Information" tab, forcing the user to hunt through all tabs until it finds the offending field.

If you believe other fields could also benefit from more immediate feedback about their wrong values, then I encourage you to add `onchange` validators for them as well.